stable/sumologic-fluentd/templates/daemonset.yaml

```diff
@@ -1,3 +1,4 @@
+# Sumologic collector URL is required
```

Comment:
I'm assuming we couldn't use the `required` template function since it would start failing the same way as before?

Reply:
we can not, and we can't use a fake value because fluentd actually tries to check if its valid